backends/smt2/smtbmc.py

```diff
@@ -1373,7 +1373,7 @@ def smt_check_sat(expected=["sat", "unsat"]):
             smt_push()
             smt_assert("(distinct (covers_%d s%d) #b%s)" % (coveridx, step, "0" * len(cover_desc)))
 
-            if smt_check_sat() == "unsat":
+            if smt_check_sat() != "sat":
```

Comment:
Hell NO! When the solver returns say `unknown` or `interrupted` we _must not_ treat it the same as if that would be `unsat`. Unsat is a guarantee that no solution exists, which is very different from say `unknown` or `interrupted`.